js/components/common/react_modal.js.jsx

```diff
@@ -311,6 +251,69 @@ class CourseModal extends React.Component {
 //Use React component from search.js
 class Description extends React.Component {
   render() {
+    const timetableUnavailable =
+      Object.keys(this.props.sessions).length !== 0 &&
+      this.props.sessions.F.length === 0 &&
+      this.props.sessions.S.length === 0 &&
+      this.props.sessions.Y.length === 0
+
+    let timetableDisplay
+    if (timetableUnavailable) {
+      timetableDisplay = (
+        <div className="timetable-unavailable-warning">
+          <div>
+            <h5 className="warning-title">
```

Comment:
Overall, let's go for a more neutral message here. You seem to be approaching this task with the idea that if timetable information is missing, there's an error. This isn't the intended case: it is perfectly normal for a course not to be offered in an academic year, which would make having not timetable information the expected result.

So, remove the red background and just have a simple message here: "No timetable information available." Don't refer to the A&S timetable, since we would not normally expect any additional information there that isn't already in Courseography. (Remember that I said on Slack that any errors in parsing timetable data can be fixed separately, but should not impact what you are doing here.)